content/docs/csidriver/installation/test/certcsi.md

```diff
@@ -7,52 +7,137 @@ description: Tool to validate Dell CSI Drivers
 Cert-CSI is a tool to validate Dell CSI Drivers. It contains various test suites to validate the drivers. 
 
 ## Installation
-To install this tool you can download one of binary files located in [RELEASES](https://github.com/dell/cert-csi/releases)
 
-You can build the tool by cloning the repository and running this command:
+The recommended methods of installing `cert-csi` are downloading the exectuble from the latest GitHub Release or pulling the latest container image.
+
+> The exectuable from the GitHub Release only supports Linux. For non-Linux users, you must build the `cert-csi` executable [locally](#building-locally).
+
+### Download Release (Linux)
+
+1. Download the latest release of the cert-csi zip file.
+
 ```bash
-make build 
+curl -LO https://github.com/dell/cert-csi/releases/download/v1.3.0/cert-csi-v1.3.0.zip
+```
+
+2. Unzip the file.
+
+``` bash
+unzip cert-csi-v1.3.0.zip
+chmod +x ./cert-csi-v1.3.0
 ```
 
-You can also build a docker container image by running this command:
+3. Install cert-csi-v1.3.0 as cert-csi.
+
 ```bash
-make docker
+sudo install -o root -g root -m 0755 cert-csi-v1.3.0 /usr/local/bin/cert-csi
 ```
 
-If you want to collect csi-driver resource usage metrics, then please provide the namespace where it can be found and install the metric-server using this command (kubectl is required):
+If you do not have root access on the target system, you can still install cert-csi to the ~/.local/bin directory:
 
 ```bash
-make install-ms
+chmod +x cert-csi-v1.3.0
+mkdir -p ~/.local/bin
+mv ./cert-csi-v1.3.0 ~/.local/bin/cert-csi
+# and then append (or prepend) ~/.local/bin to $PATH
 ```
-[FOR UNIX] If you want to build and install the tool to your $PATH and enable the **auto-completion** feature, then run this command:
+
+### Pull The Container Image
+
+   {{< tabs name="pulling-cert-csi-image" >}}
+   {{% tab name="Docker" %}}
+
+   ```bash
+      docker pull dellemc/cert-csi:v1.3.0
+   ```
+
+   {{% /tab %}}
+   {{% tab name="Podman" %}}
+
+   ```bash
+      podman pull dellemc/cert-csi:v1.3.0
```

Comment:
Your call but do we really need to put the `podman` command ?

This is pretty much equivalent and everybody who uses `podman` knows

Reply:
It's redundant but from feedback from users, it's better to be explicit.